Northstar.Client/mod/scripts/vscripts/client/cl_titan_cockpit.nut

```diff
@@ -1016,14 +1034,24 @@ string function RollRandomEjectString()
 	float randForType = RandomFloat( 1.0 )
 	if ( randForType < CHANCE_FOR_RARE )
 	{
-		int index = RandomInt( COCKPIT_EJECT_RARE_COUNT )
-		string result = "#COCKPIT_EJECT_RARE_" + index
-		return result
+		int index = RandomInt( COCKPIT_EJECT_RARE_COUNT + file.moddedRareEjectMessages.len() )
+		if ( index <= COCKPIT_EJECT_RARE_COUNT )
+		{
+			return "#COCKPIT_EJECT_RARE_" + index
+		}
+		else
+		{
+			return file.moddedRareEjectMessages[index - COCKPIT_EJECT_RARE_COUNT]
+		}
```

Comment:
In the event that `index == COCKPIT_EJECT_RARE_COUNT` this code will try to return an ejection message that doesn't exist (`"#COCKPIT_EJECT_RARE_36"`) instead of returning `file.moddedRareEjectMessages[0]`
This also means that `file.moddedRareEjectMessages[0]` will never be used.

![image](https://user-images.githubusercontent.com/66967891/201214782-2c86d347-33df-437f-ac7f-e8e135a304c1.png)


```suggestion
		if ( index < COCKPIT_EJECT_RARE_COUNT )
			return "#COCKPIT_EJECT_RARE_" + index
		else
			return file.moddedRareEjectMessages[index - COCKPIT_EJECT_RARE_COUNT]
```
Also {} isnt needed for single line things, neither is the else technically but personally I prefer to keep it for better readability